### DISCUSSION TITLE
Tech Stack - Confirmation

### DISCUSSION THREAD

**Discussion by prafullmaru** [prafullmaru/ePanchayat#2] [General]
We have planned with below technical stack at major:
- Angular framework for front-end (With HTML and CSS)
- .NET 6.0 for RESTful API
- Microsoft SQL for data storage

Authentication part is yet to finalize. Let us know if there is any suggestions.

**Comment by umesh23** [comment]
We can think on to use Auth0.

**Comment by Narigara007** [comment]
I guess, you need to think about mobile apps development using Auth0 authentication. you can also use open source React Native framework for mobile apps development which supports iOS and Android both the platform using single code base.
